internal/incident/incident.go

```diff
@@ -29,17 +31,30 @@ type Incident struct {
 	Rules           map[ruleID]struct{}
 	Recipients      map[recipient.Key]*RecipientState
 
-	incidentRowID int64
+	Icingaweb2Url string
```

Comment:
That `Icingaweb2Url` doesn't make much sense here. It's passed as an argument to `(*Incident).ProcessEvent()` which then sets this member, then calls `(*Incident).notifyContacts()` which then accesses the member to pass it as an argument to other functions.

Either continue the URL as a parameter or add a member for the full config (and initialize that one in a proper place rather than as a side effect of `ProcessEvent()`).